examples/magento-graphcms/pages/checkout/success.tsx

```diff
@@ -56,7 +56,13 @@ function OrderSuccessPage() {
               <Trans id='Thank you for your order!' />
             </LayoutTitle>
             <CartSummary />
-            <CartItemSummary />
+            <CartItemSummary
+              sx={{
+                '& .Scroller-root.grid': {
+                  display: 'flex',
+                },
+              }}
+            />
```

Comment:
Shall we move this to inside the CartItemSummary, because I think it is a problem for all shops